### ISSUE TITLE
Windows: The gateway is not from external card

### ISSUE THREAD

**Issue by koraniar** [silverwind/default-gateway#15]
When a computer has virtual network adapters with a gateway, it's selecting the first that finds, but not necessarily is the gateway of the external card.

Example: after run `wmic path Win32_NetworkAdapterConfiguration where IPEnabled=true get DefaultIPGateway,Index /format:table` in windows I get this response:
```
3 
{"0.0.0.0"} 5 
{"192.168.1.1", "fe80::dafb:5eff:fe33:3489"} 6 
8
```

and after parse it the result is `0.0.0.0`, but the gateway of the external card is `192.168.1.1`

**Comment by silverwind** [comment]
How does this configuration look in the GUI? I take it there are multiple default gateways in this window?

![](https://user-images.githubusercontent.com/115237/59901875-ed130880-93fc-11e9-8ffe-4ff5421750ca.png)

**Comment by silverwind** [comment]
Fixed in `default-gateway@5.0.3`.
